Project/js/content.js

```diff
@@ -0,0 +1,94 @@
+const b = new URL('https://api.citybik.es/v2/networks');
+const w = new URL("https://simple.wikipedia.org/w/api.php?action=opensearch&origin=*");
```

Comment:
these are some criptic names of variables. Please make variables, functions, classes etc. as readable as possible